cprover_bindings/src/irep/to_irep.rs

```diff
@@ -347,7 +360,15 @@ impl ToIrep for Parameter {
 
 impl ToIrep for Stmt {
     fn to_irep(&self, mm: &MachineModel) -> Irep {
-        self.body().to_irep(mm).with_location(self.location(), mm)
+        // Match Assert in Stmt to add fields to Source Location Irep
```

Comment:
Needs better comment to explain what's going on here.

Reply:
sure